packages/core/certora/optimistic-asserter/specs/Asserter_Auxiliary.spec

```diff
@@ -0,0 +1,186 @@
+// For every new spec you create, import this basic spec file:
+import "./Asserter_Base.spec"
+import "./dispatchedMethods.spec"
+
+using TestnetERC20 as testERC20
+/**************************************************
+ *      Top Level Properties / Rule Ideas         *
+ **************************************************/
+// Here you can write your ideas for rules and implement them 
+// in this spec.
+
+/**************************************************
+ *                  Misc. rules                   *
+ **************************************************/
+
+// A simple rule that checks which of the main contract methods
+// are reachable (reach the assert false statement after function call).
+ rule sanity(method f) {
+    env e;  // Environment variable - includes all transaction and block information.
+    calldataarg args; // arbitrary calldata - adapted for every method signature implicitly
+    f(e, args);
+    assert false;
+}
+
+// Checks that view functions in the contract never revert. 
```

Comment:
Is this a common check? I would expect nontrivial view functions to often revert under _some_ conditions (like weird contract states or input overflows).

Reply:
It's not very common, but it's helpful to be aware of a state that gets some view functions to revert.
Overflow is definitely an option.
Sometimes we use view functions to get values from storage, so if they revert, we need to bear in mind that we kind of filter the reverting states of the same storage.